pkg/diskmaker/diskmaker.go

```diff
@@ -310,3 +341,95 @@ func fileExists(filename string) bool {
 	}
 	return true
 }
+
+// getPVCreationLock checks whether a PV can be created based on this device
+// and Locks the device so that no PVs can be created on it while the lock is held.
+// the PV lock will fail if:
+// - another process holds an exclusive file lock on the device (using the syscall flock)
+// - a symlink to this device exists in symlinkDirs
+// returns:
+// ExclusiveFileLock, must be unlocked regardless of success
+// bool determines if flock was placed on device.
```

Comment:
I know we return a bool indirectly via the exclusiveFileLock, but I wonder if this was part of a prior solution? Perhaps the comment here should be updated?